Add info for new mailing list to documentation. (#1607)
ESMValGroup/ESMValTool: doc/sphinx/source/community/contact.rst

Contacts for specific diagnostic sets are the respective authors, as
listed in the corresponding diagnostic documentation and in the source
code.

.. _mailing-list:

User mailing list
-----------------

> The ESMValTool user mailing list is open for all general and technical questions on the ESMValTool such as questions on installation, application or development, etc. To subscribe send an email to `sympa@listserv.dfn.de <mailto:sympa@listserv.dfn.de?subject=subscribe%20esmvaltool>`_ with the following subject line:

Reviewer: ```suggestion
The ESMValTool user mailing list is open for all general and technical questions on the ESMValTool, for example about installation, application, development, etc.
To subscribe send an email to `sympa@listserv.dfn.de <mailto:sympa@listserv.dfn.de?subject=subscribe%20esmvaltool>`_ with the following subject line:
```
